### DISCUSSION TITLE
MAC Binding LTSP Client

### DISCUSSION THREAD

**Discussion by perlland** [ltsp/ltsp#566] [General]
Can it be possible to MAC bind LTSP Client machine? My motive is to restrict unauthorised PXE boot access to my LTSP Server.
Thanking You in advance

**Comment by alkisg** [comment]
Sure, read this dnsmasq option:
https://manpages.debian.org/bullseye/dnsmasq-base-lua/dnsmasq.8.en.html#G,

I.e. you can use `dhcp-hosts` in dnsmasq to list your clients, and `dhcp-ignore=tag:!known` to ignore the unknown ones.

I've also written [detailed instructions for this in Greek](https://ts.sch.gr/wiki/Linux/LTSP/%CE%A0%CF%81%CE%BF%CF%87%CF%89%CF%81%CE%B7%CE%BC%CE%AD%CE%BD%CE%B1/%CE%A0%CE%BF%CE%BB%CE%BB%CE%AC_%CE%B5%CF%81%CE%B3%CE%B1%CF%83%CF%84%CE%AE%CF%81%CE%B9%CE%B1#.CE.A0.CE.BF.CE.BB.CE.BB.CE.BF.CE.AF_.CE.B5.CE.BE.CF.85.CF.80.CE.B7.CF.81.CE.B5.CF.84.CE.B7.CF.84.CE.AD.CF.82_LTSP), maybe you could use them via google-translate.
